src/utils/numbers.js

```diff
@@ -47,7 +47,7 @@ export function toFixed(x) {
 }
 
 export const calculateMaxBorrow = (creditLimit, originationFee) => {
-  const cl = Number(creditLimit.mul(99999).div(100000).toString());
+  const cl = Number(creditLimit.mul(9999).div(10000).toString());
```

Comment:
So I think the solution here is do this as `9990`. And then add `getCreditLimit` to `usePollMemberData` so then it shouldn't go as far out of date.